1.8/test/run

```diff
@@ -1,131 +1,172 @@
 #!/bin/bash
 #
-# The 'run' script performs simple tests that verifies usability
-# of tools, packaged in toolchain image.
+# The 'run' performs a simple test that verifies that S2I image.
+# The main focus here is to excersise the S2I scripts.
 #
 # IMAGE_NAME specifies a name of the candidate image used for testing.
 # The image has to be available before this script is executed.
 #
-# DEBUG environment variable, if not empty, makes 'run' to log every step
-# of testing.
-#
+export IMAGE_NAME=${IMAGE_NAME:-sclorg/python-36-centos7-candidate}
```

Comment:
No python image and sclorg/ prefix. So default could be `centos/golang-18-centos7`.

Reply:
Remnants of my inspiration :). Will fix.